GROOMME.md

```diff
@@ -0,0 +1 @@
+Please groom the changelog and then delete me.
```

Comment:
Can we delete this now that is has been groomed?

Reply:
yes forgot that step will delete